apps/api/src/app/events/usecases/add-job/add-job.usecase.ts

```diff
@@ -69,6 +89,10 @@ export class AddJob {
   }
 
   public static toMilliseconds(amount: number, unit: DigestUnitEnum): number {
+    Logger.debug('Amount is: ' + amount);
+    Logger.debug('Unit is: ' + unit);
```

Comment:
What do you think about combining those log to one execution?

Reply:
If we do that then new relic will parse the values together instead of apart.